### ISSUE TITLE
Logrus is in maintenance mode

### ISSUE THREAD

**Issue by miparnisari** [mailgun/gubernator#210]
See https://github.com/sirupsen/logrus/blob/dd1b4c2e81afc5c255f216a722b012ed26be57df/README.md?#L6-L8

Consider moving to `zap`: https://github.com/uber-go/zap

**Comment by thrawn01** [comment]
Agreed! Logrus wasn't a priority since it works, and if we move to something else, it should probably be the new standard golang logger. https://go.dev/blog/slog
